src/test/suite/extensionFunctions.test.ts

```diff
@@ -152,7 +152,7 @@ suite('Extension Functions Test Suite', () => {
 	});
 
 	test('try executing a valid command', async function() {
-		await extensionFunctions.cliExecutionCheck('test-pwd','pwd').then( (returnBool) => {
+		await extensionFunctions.cliExecutionCheck('test-pwd','echo').then( (returnBool) => {
```

Comment:
shouldn't `test-pwd` renamed `test-echo`?

Reply:
Spun off https://issues.redhat.com/browse/FUSETOOLS2-915 to investigate deeper command testing on other platforms